scripts/ci/kubernetes/docker/airflow-test-env-init.sh

```diff
@@ -19,7 +19,7 @@
 
 set -x
 
-cd /usr/local/lib/python2.7/dist-packages/airflow && \
+cd /usr/local/lib/python3.6/dist-packages/airflow && \
```

Comment:
It looks like this path should use `python3.5` instead of `python3.6`. Xenial LTS installs `python3.5` unless we add a new PPA.

Reply:
Thanks for catching. But I change base docker images to python:3.6-slim because we have a jira ticket in https://issues.apache.org/jira/browse/AIRFLOW-4210. And I think we should change our ci k8s as well.